docs/application/native/guides/compliations/watchface-complication.md

```diff
@@ -152,7 +146,7 @@ void _watchface_complication_updated_cb(int complication_id,
 ```
 
 
-If the callback is not used any more, it should be be removed by using watchface_complication_remove_updated_cb function.
+If the callback is not used any more, it should be be removed by using `watchface_complication_remove_updated_cb()`:

```

Comment:
Please change as"
If the callback is not used any more, it must be removed by using `watchface_complication_remove_updated_cb()`:

Word "be" is used twice.